release-notes/docs/3.9.0.md

```diff
@@ -15,6 +15,15 @@ None.
 
 ## Upgrade notes
 
+### CHT Docker Image
+
+#### 3.9 CHT Docker Image released
```

Comment:
Let's make it more obvious that this is required and not optional...

```suggestion
#### Docker image update required
```

Also can we move this whole section down into "breaking changes"?